core-java-modules/core-java-datetime-string/src/test/java/com/baeldung/datetime/DateTimeFormatterUnitTest.java

```diff
@@ -159,4 +160,33 @@ public void shouldExpectAnExceptionIfDateTimeStringNotMatchPattern() {
         DateTimeFormatter zonedFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm z");
         ZonedDateTime.from(zonedFormatter.parse("31.07.2016 14:15"));
     }
+
+    @Test
+    public void shouldPrintFormattedZonedDateTime() {
+        ZonedDateTime zonedDateTime = ZonedDateTime.of(2021, 02, 15, 0, 0, 0, 0, ZoneId.of("Europe/Paris"));
+        String formattedZonedDateTime = DateTimeFormatter.ISO_INSTANT.format(zonedDateTime);
+        Assert.assertEquals(formattedZonedDateTime, "2021-02-14T23:00:00Z");
```

Comment:
swap the order of arguments so that it's assertEquals(expected, actual) -- do this for the other test that uses assertEquals as well. And for both, add an empty line before the assertion.

Reply:
Done